src/app/components/normal-component/confirm-inscription/confirm-inscription.component.spec.ts

```diff
@@ -22,14 +14,15 @@ describe('ConfirmInscriptionComponent', () => {
                 ConfirmInscriptionComponent,
             ],
             providers: [
-                { provide: AuthenticationService, useValue: authenticationServiceStub },
+                { provide: AuthenticationService, useClass: AuthenticationServiceMock },
             ],
         }).compileComponents();
+        AuthenticationServiceMock.setUser(AuthenticationServiceMock.CONNECTED);
         fixture = TestBed.createComponent(ConfirmInscriptionComponent);
         component = fixture.componentInstance;
     });
     it('should create and send notification', () => {
-        spyOn(component.authService, 'sendEmailVerification').and.callFake(async() => {});
+        spyOn(component.authService, 'sendEmailVerification');
```

Comment:
Par défaut ça fait "and call through" ?

Reply:
Par défaut il semblerait que ça fasse pas de callThrough justement !